### PR TITLE
Edl 500 error bug

### PR DESCRIPTION
**Pull request by MosheEichler**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/36876

## Description
change in the ***is_demisto_version_ge*** function to compare the build_number as two numbers instead of two strings
## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 5.5.0
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/Base/Scripts/CommonServerPython/CommonServerPython_test.py [hunk 1]
```diff
@@ -1203,6 +1203,26 @@ def test_is_demisto_version_build_ge(mocker):
...
+    """
+    mocker.patch.object(
+        demisto,
+        'demistoVersion',
+        return_value={
+            'version': '6.1.0',
+            'buildNumber': '999999'
+        }
+    )
+    assert not is_demisto_version_ge('6.1.0', build_number='1000000')
```
Comment (reutshal) [inline Packs/Base/Scripts/CommonServerPython/CommonServerPython_test.py:1223]: Why not add to the previous UT an assertion like this:

assert not is_demisto_version_ge('6.0.0', '400000')

**Review by reutshal — CHANGES_REQUESTED** [review]
please see my comment regarding the test

File: Packs/Base/Scripts/CommonServerPython/CommonServerPython_test.py (hunk 1 above)
Comment (MosheEichler) [reply Packs/Base/Scripts/CommonServerPython/CommonServerPython_test.py]: I remove this function and I added to the previous function, this case ***is_demisto_version_ge('6.0.0', '6')***, That in strings comparison it's bigger than build_number '500000' but in integers comparison it's lower

File: Packs/Base/Scripts/CommonServerPython/CommonServerPython_test.py [hunk 2]
```diff
@@ -1201,6 +1201,7 @@ def test_is_demisto_version_build_ge(mocker):
     assert not is_demisto_version_ge('6.0.0', '50001')
     assert not is_demisto_version_ge('6.1.0', '49999')
     assert not is_demisto_version_ge('5.5.0', '50001')
+    assert is_demisto_version_ge('6.0.0', '6')
```
Comment (reutshal) [inline Packs/Base/Scripts/CommonServerPython/CommonServerPython_test.py:1204]: Please move to line 1201 to keep consistency.
I would also add a comment explaining this assertion, something like - 'Added with the fix of __link to the issue__ '

**Review by reutshal — APPROVED** [review]

**Pull request merged**